Extension/c_cpp_properties.schema.json

```diff
@@ -58,6 +58,10 @@
                             "type": "string"
                         }
                     },
+                    "windowsSDKVersion": {
+                        "description": "Version of the Windows SDK include path to use on Windows, e.g. '10.0.17134.0'.",
+                        "type": "string"
```

Comment:
should probably add `patternProperties` to this so we can get squiggles.  (see the `env` property).

The regexp is this, I think: `^\d{2}\.\d{1}\.\d{5}\.\d{1}$`

Reply:
Ok.